### ISSUE TITLE
Octos Behaves like a fullscreen application instead of a Desktop replacement

### ISSUE THREAD

**Issue by Phoenix1o1** [underpig1/octos#4]
Upon Reinstall due to AV deleting some files, with an exception added on all impacted folders, the application behaves as a fullscreen program instead of replacing the desktop.
Please find attached a video of the problem

https://github.com/underpig1/octos/assets/5053221/9b3fa352-960e-4372-9cd9-e961ddf0005b

**Comment by srsng** [comment]
Yes it is.And you can find it consumes a lot of CPU performance(2% of my i12700h). But it works,.
